inf-terraform-azure/README.md

```diff
@@ -0,0 +1,6 @@
+# Azure Quickstarter (inf-terraform-azure)
+
+Documentation is located in our [official documentation](https://www.opendevstack.org/ods-documentation/ods-quickstarters/latest/index.html)
+
+Please update documentation in the [antora page directory](https://github.com/opendevstack/ods-quickstarters/tree/master/docs/modules/ROOT/pages)
```

Comment:
We have to update this file: ods-quickstarters/master/docs/modules/quickstarters/pages/index.adoc
It lacks also a toc entry for AWS.